modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaResteasyServerCodegen.java

```diff
@@ -9,10 +10,11 @@
 import java.io.File;
 import java.util.*;
 
-public class JavaResteasyServerCodegen extends AbstractJavaJAXRSServerCodegen implements JbossFeature {
+public class JavaResteasyServerCodegen extends AbstractJavaJAXRSServerCodegen implements JbossFeature, BeanValidationFeatures {
 
+	protected boolean useBeanValidation = true;
```

Comment:
Cleaned the tabs, I set useBeanValidation=true because @cbornet suggested to do so (i.e. BeanValidation is a basic feature every language should support automatically). Of course we can also set to false, please advise what is preferred?